ui/appui-react/src/appui-react/UiFramework.ts

```diff
@@ -91,6 +96,96 @@ export interface TrackingTime {
  * @public
  */
 export class UiFramework {
+  /**
```

Comment:
For reviewers, I suggest starting here which is the root...

From here, you can look at the files exporting the `Internal*` classes, I'll have to extract each API instead of exposing the internal here, but other than that, the idea are there.

The rest of the files are mostly reacting to these changes, and trying not to use the deprecated stuff. (The idea is that eventually the implementation of the UiFramework properties would not be static classes but either instances where it make sense, or simply namespace, but only accessible through this single entry point)